docs/docs/cloud/cloud-cli.mdx

```diff
@@ -26,7 +32,7 @@ If you are unsure of the current value of an env var, use the `set` command to s
 The `list` subcommand provides an interface to see existing set env var keys:
 
 ```sh
-meltano-cloud config env list --limit 5
+meltano cloud config env list --limit 5
```

Comment:
I vaguely remember some discussion that we are deprecating meltano cloud for meltano-cloud, do we need to update all these docs to use meltano-cloud?

Reply:
Probably, I think the plan is to eventually split `meltano-cloud` away from meltano core entirely and have it be its own standalone CLI app but @tayloramurphy could clarify or correct me on that.

Reply:
So we don't want to pull the removal of the `-` part of the command into `main`.  It looks like there's a copy of the cloud cli docs in the reference section as well. 

I mainly want to make sure the notifications and add projects updates land.